addons/binding/org.openhab.binding.boschindego/deindego.map

```diff
@@ -0,0 +1,23 @@
+0=Status lesen
```

Comment:
This is for transforming the numeric states to text readable for the users. They are not used by the binding. They are for the users who are using MAP for transforming. I said this in the README file. They can use this files as template to create their own map files in their native language.